lib/std/Progress.zig

```diff
@@ -143,6 +147,7 @@ pub fn start(self: *Progress, name: []const u8, estimated_total_items: usize) !*
         self.terminal = stderr;
         self.supports_ansi_escape_codes = true;
     } else if (std.builtin.os.tag == .windows and stderr.isTty()) {
+        self.is_windows_terminal = true;
```

Comment:
How do you get here? `isCygwinPty` and `supportsAnsiEscapeCodes` should detect this case and use escape sequences instead of WinAPI functions.

Reply:
Yes, that is the case. I was just trying to make it more explicit. The first case of the if/else if chain there is:

`if (stderr.supportsAnsiEscapeCodes()` which is just `isCygwinPty()` on windows.

I didn't add a comment but `supports_ansi_escapse_codes` and `is_windows_terminal` are mutually exclusive.